design/validator_info.md

```diff
@@ -65,9 +65,35 @@ This file is updated by node once a minute and contains following information:
 ```
 
 ## Modification - New Read Command
-Validator_info should be accessible as read command, available for TRUSTEE and STEWARDS only. New command should provide info from
-all the connected nodes without need of consensus (similar to force=True flag in upgrade cmd). Command should allow
-requesting all parameters or some subset of parameters.
+Validator_info accessible as read command, available for all clients. New command provide info from
+all the connected nodes without need of consensus (similar to force=True flag in upgrade cmd).
+Command allow requesting all parameters or some subset of parameters.
+
+The client sends a command with some parameters to each node. There are only one parameter now - node alias for get its info. But parameters list will expanded later.
+After receiving the request, each node starts a validator-info script, and then sends the Json result without compression to the client.
+The client should not wait for the consensus of the all node, but should handle the response from each node separately.
+
+Request for getting validator info data :
```

Comment:
Please mention new commands (restart and validator info) in https://github.com/hyperledger/indy-node/blob/master/docs/requests.md

Reply:
done